### ISSUE TITLE
Keys stick sometimes in custom keybindings

### ISSUE THREAD

**Issue by sevagh** [ueokande/vim-vixen#696]
I'm using vim-vixen (the latest) with `Mozilla Firefox 71.0` on Linux, Fedora 30.

I've added some custom bindings e.g.:
```
    "x": { "type": "tabs.close" },
    "J": { "type": "tabs.prev" },
    "K": { "type": "tabs.next" },
```
Now when I press `x`, `J`, or `K`, the behavior will repeat - e.g. `x` instead of closing a tab will close every single tab until Firefox exits.

Or, `J` will put me in an infinite `tabs.prev` loop, cycling through every single tab until I exit Firefox.

This broke during a Firefox upgrade - I'm trying to figure out which one, based on my previous upgrade history. Is this anything seen before, any ideas how I could solve it?

**Comment by sevagh** [comment]
It can also just repeat for a low amount of times (so, `K` will move 3 tabs to the right instead of 1), or most of the time work normally.

**Comment by sevagh** [comment]
I downgraded my Fedora firefox version from `71.0-15.fc30` to `70.0.1-4.fc30`, the last good version I had (downloaded it from Koji), and things are back to normal, no key duplication.

**Comment by sevagh** [comment]
I'm now testing vim-vixen 0.25 - https://addons.mozilla.org/en-US/firefox/addon/vim-vixen/versions/

**Comment by AlexeySachkov** [comment]
@sevagh, could you please check if this is the same as #693? 

I.e. you press some number and then use some keybinding and it gets repeated several times

**Comment by sevagh** [comment]
I don't intentionally press numbers but it could be related. E.g. when I use "Alt+n" to jump across several tabs, and then another keybinding ("x" to close). But the strange thing is that sometimes the keybindings repeat forever. I'll press "J" (go to previous tab) and it will cycle through tabs endlessly, consuming 100% CPU until I quit Firefox.

Either way, downgrading to vim-vixen 0.25 got rid of the problem. I can try 0.27 when it's ready.

**Comment by fbarthelery** [comment]
I have the same issue on vim-vixen-0.26. It's definitelly not the same as #693 as I'm sure I'm not hitting any number keys when it happened. 
I'll downgrade to 0.25, and report back if fix it for me also

**Comment by cdepillabout** [comment]
@fbarthelery Did downgrading fix this for you?  I'm seeing the same thing, and it is really annoying losing a bunch of open tabs.

Alternatively, I'd be happy with any other vim-keybinding plugin for firefox as long as it works well.  I've been pretty happy with vim-vixen up until now...

**Comment by fbarthelery** [comment]
It seems to fix the issue but I only run it for a few days

**Comment by Eug-VS** [comment]
I have exactly the same problem as @sevagh has. Thanks for the temporary solution!

**Comment by ueokande** [comment]
It might same as #693 and I'll fix the issue on v0.27.  Here is a reproduce steps:

1. Open two tabs and select a first tab
1. Press, <kbd>Alt</kbd>+<kbd>9</kbd>, <kbd>Alt</kbd>+<kbd>1</kbd>, <kbd>Alt</kbd>+<kbd>9</kbd>, and <kbd>Alt</kbd>+<kbd>1</kbd>
1. Press  <kbd>Shift</kbd>+<kbd>J</kbd>

Then, the add-on switches tabs 99 times!

**Comment by ueokande** [comment]
fixed in  #709
